src/languages/html.coffee

```diff
@@ -0,0 +1,681 @@
+helper = require '../helper.coffee'
+parser = require '../parser.coffee'
+
+parse5 = require '../../vendor/parse5/index.js'
```

Comment:
Now that the build is browserified, can you just add parse5 to package.json and then use require 'parse5'?